doc/admin_guide.rst

```diff
@@ -19,48 +19,28 @@ installed the following software of framework in the machine:
 -  Rule engine dependencies:
 
    -  Python 2.7.6 `[1] <http://www.python.org/download/releases/2.7.6/>`_
-   -  PyClips 1.0 `[2] <http://sourceforge.net/projects/pyclips/files/>`_
```

Comment:
You need to have installed PyClips in order to use the Rule Engine, why do you delete it?